project/Settings.scala

```diff
@@ -135,6 +135,18 @@ object Settings {
     libraryDependencies ++= databaseMigrationDependencies
   ) ++ commonSettings
 
+  val cromwellApiClientSettings = List(
+    name := "cromwell-api-client",
+    libraryDependencies ++= cromwellApiClientDependencies,
+    organization := "org.broadinstitute",
+    scalaVersion := "2.12.1",
```

Comment:
I assume that'll only work if the dependencies are also present in 2.11? 

I had to do a little dance with the version numbers so I'm not 100% sure how easy this'll be